packages/common/src/store/pages/saved-page/actions.ts

```diff
@@ -84,7 +83,7 @@ export const fetchSavesRequested = () => ({
   type: FETCH_SAVES_REQUESTED
 })
 
-export const fetchSavesSucceeded = (saves: Favorite[]) => ({
+export const fetchSavesSucceeded = (saves: any[]) => ({
```

Comment:
Are there conflicting `Favorite` types, or are they not just favorites anymore cause of the new filters?